examples/webgl_multiple_rendertargets_read.html

```diff
@@ -0,0 +1,299 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - Multiple Render Targets Read Pixels</title>
```

Comment:
Instead of creating a new example, do you mind enhancing `webgl_read_float_buffer` instead?

Reply:
Sure, do you mean changing the target there to be with multiple textures or adding an option in the UI to switch between single and multiple?

Reply:
I think we can just use MRT.